lib/internal/util.js

```diff
@@ -250,8 +248,19 @@ function getConstructorOf(obj) {
   return null;
 }
 
+let uvBinding;
+function lazyErrmapGet(name) {
+  if (!uvBinding) {
+    uvBinding = internalBinding('uv');
+  }
+  if (!uvBinding.errmap) {
+    uvBinding.errmap = uvBinding.getErrorMap();
+  }
```

Comment:
Suggestion: move this if inside the one above. That way it's one check less for each function call.

Reply:
@BridgeAR That will recreate the ermap the call to `lazyErrmapGet` in `lib/internal/util.js` happens to be later than the call to `lazyErrmapGet` in `lib/internal/errors.js`, though, and I don't think there is any guarantee about the order?

Reply:
Oh, my bad.